src/DAL/index.ts

```diff
@@ -15,15 +18,28 @@ const pool = new Pool({
 });
 
 pool.on('connect', () => {
-    console.log('initiated db connection pool');
+    logger.info({
+        step: 'initiated db connection pool',
+        severity: Severity.LOW
+    });
 });
 
 export const deleteOldInvestigations = async (): Promise<void> => {
     try {
-        console.log('executing DB query to delete investigations');
+        logger.info({
+            step: 'executing DB query to delete investigations',
+            severity: Severity.LOW
+        });
         await pool.query(deleteInvestigationsQuery(subDays(new Date(), +process.env.MIN_DAYS_TO_REMOVE), countiesBlackList));
-        console.log('finished deleting investigations successfully');
+        logger.info({
+            step: 'finished deleting investigations successfully',
+            severity: Severity.LOW
+        });
     } catch (error) {
+        logger.error({
+            step: 'got error in querying the DB, throwing an exception',
```

Comment:
add the error to the log

Reply:
the error is being printed afterwards in the catch block that catches the exception

Reply:
i think one log about the error is enoughl, combinw them or leave only one of them

Reply:
Done, removed this log